test/unit/js/WebsocketControllerTests.js

```diff
@@ -42,6 +42,7 @@ describe('WebsocketController', function () {
       leave: sinon.stub()
     }
     return (this.WebsocketController = SandboxedModule.require(modulePath, {
+      globals: { console },
```

Comment:
Is this needed or was it from debugging?

Reply:
For debugging only -- removed in https://github.com/overleaf/real-time/pull/162/commits/e157a1bc6ac6c86dce78065464bed81eb35d8eaf